docs/src/use/core-concepts/glossary.md

```diff
@@ -0,0 +1,396 @@
+
+You can let ESLint know which global variables your code uses with [global declarations](#global-declaration).
+
+## I
+
+### Inline Config (Configuration Comment)
+
+A source code comment that configures a rule to a different severity and/or set of options.
+
+Inline configs use similar same syntax as [config files](#config-file-configuration-file) to specify any number of rules by name, their new severity, and optionally new options for the rules.
+For example, the following inline config comment simultaneously disables the `eqeqeq` rule and sets the `curly` rule to `"error"`:
+
+```js
+/* eslint eqeqeq: "off", curly: "error" */
+```
+
+For documentation on inline config comments, see [Rules > Using configuration comments](../configure/rules#using-configuration-comments).
+
+## L
+
+### Legacy Config
+
+The previous configuration file format for ESLint, now superseded by ["Flat" config](#flat-config).
+
+Legacy ESLint configurations are named in the format `.eslintrc.*` and allowed to be nested across files within sub-directories in a project.
+
+You can read more about the lifetime of legacy configurations in [ESLint's new config system, Part 1: Background](https://eslint.org/blog/2022/08/new-config-system-part-1).
+
+### Linter
+
+A [static analysis](#static-analysis) tool that can [report](#report) the results from running a set of [rules](#rule) on source code.
+Each rule may report any number of [violations](#violation) in the source code.
+
+ESLint is a commonly used linter for JavaScript and other web technologies.
+
+Note that a _linter_ is separate from [formatters](#formatter-tool) and [type checkers](#type-checker).
+
+### Logical Rule
+
+A [rule](#rule) that inspects how code operates to find problems.
+
+Many logical rules look for likely crashes (e.g. [`no-undef`](../../rules/no-undef)), unintended behavior (e.g. [`no-sparse-arrays`](../../rules/no-sparse-arrays)), and unused code (e.g [`no-unused-vars`](../../rules/no-unused-vars)),
+
+You can see the full list of logical rules that ship with ESLint under [Rules > Possible Problems](../../rules/#possible-problems)
+
+## N
+
+### Node
+
+A section of code within an [AST](#abstract-syntax-tree-ast).
+
+Each node represents a type of syntax found in source code.
+For example, the `1 + 2` in the AST for `1 + 2;` is a _BinaryExpression_.
+
+See [#esquery](#esquery) for the library ESLint uses to parse [selectors](#selector) that allow [rules](#rule) to search for nodes.
+
+## O
+
+### Override
+
+When a [config object](#config-object) or [inline config](#inline-config-configuration-comment) sets a new severity and/or rule options that supersede previously set severity and/or options.
+
+The following [config file](#config-file-configuration-file) overrides `no-unused-expression` from `"error"` to `"off"` in `*.test.js` files:
+
+```js
+export default [
+  rules: {
+    "no-unused-expressions": "off"
+  },
```

Comment:
```suggestion
  {
    rules: {
      "no-unused-expressions": "error"
    }
  },
```

Missing object `{}`, and per the description the severity should be `"error"` (so that the next object overrides it from `"error"` to `"off"`).